core/src/main/scala/org/apache/spark/ui/UIUtils.scala

```diff
@@ -27,13 +27,16 @@ import scala.xml.transform.{RewriteRule, RuleTransformer}
 
 import org.apache.spark.internal.Logging
 import org.apache.spark.ui.scope.RDDOperationGraph
+import org.apache.commons.lang3.StringEscapeUtils
 
 /** Utility functions for generating XML pages with spark content. */
 private[spark] object UIUtils extends Logging {
   val TABLE_CLASS_NOT_STRIPED = "table table-bordered table-condensed"
   val TABLE_CLASS_STRIPED = TABLE_CLASS_NOT_STRIPED + " table-striped"
   val TABLE_CLASS_STRIPED_SORTABLE = TABLE_CLASS_STRIPED + " sortable"
 
+  val NEWLINE_AND_SINGLE_QUOTE_REGEX = "(\r\n|\n|\r|%0D%0A|%0A|%0D|'|%27)"
```

Comment:
This is just a string constant rather than regex (with `"...".r`). If you're going to do it you can make this faster by reusing the Regex object. (Can these members be private while you're at it?)